### ISSUE TITLE
suggested keybinding for swaylock conflicts with default sway config file

### ISSUE THREAD

**Issue by markstos** [Drakulix/sway-gnome#4]
I realize the idea is that people will customize the provided config file but it would be nice if the default file provided didn't conflict with the default sway file provided, which already uses both

    ` $mod+l` and `$mod+Shift+l`

**Comment by Drakulix** [comment]
Any suggestions? :)

**Comment by markstos** [comment]
I did a review of some dotfiles that included `swaylock` and the results were highly variable. There was no consensus.  I think the favorite I saw was `$mod+z`.   "Z" makes me think of "end" because it's the last letter of the alphabet. Like "end of session". So in my head I can relate that to "lock screen". 

Other ones I saw were `$mod+Esc` and `$mod+Alt+l`. 

"L" is highly used for arrow key actions, so I think it may best to avoid "L" completely. 

GNOME users "Super+L" for "Lock", so it's too bad it couldn't work here for consistency as well.
